### PR TITLE
Edit photo location

### PR DESCRIPTION
**Pull request by Swiftkage**

### CODE REVIEW

File: docs/AboutUs.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 = About Us
 :site-section: AboutUs
 :relfileprefix: team/
-:imagesDir: images/team-members
+:imagesDir: images
```
Comment (dan-delion) [inline docs/AboutUs.adoc]: Why is this changed?

Comment (Swiftkage) [reply docs/AboutUs.adoc]: From the website:
The photo of a team member should be doc/images/githbub_username_in_lower_case.png e.g. docs/images/damithc.png. If you photo is in jpg format, name the file as .png anyway.

Comment (dan-delion) [reply docs/AboutUs.adoc]: Oh, got it
